.github/workflows/pull_request.yaml

```diff
@@ -19,17 +19,17 @@ jobs:
     - uses: actions/checkout@v3
       with:
         fetch-depth: 0
-    - name: Setup GCloud Auth
-      uses: google-github-actions/auth@v1
+    - name: Set up Python
+      uses: actions/setup-python@v4
       with:
-        project_id: ${{ secrets.GCP_PROJECT_ID }}
-        credentials_json: ${{ secrets.GCP_SA_KEY }}
-        create_credentials_file: true
-        export_environment_variables: true
-    - name: Setup GCloud SDK
-      uses: google-github-actions/setup-gcloud@v1.1.0
+        python-version: 3.8
+    - uses: google-github-actions/setup-gcloud@a48b55b3b0eeaf77b6e1384aab737fbefe2085ac
       with:
-        version: '>= 386.0.0'
+        version: '290.0.1'
+        project_id: ${{ secrets.GCP_PROJECT_ID }}
+        service_account_key: ${{ secrets.GCP_SA_KEY }}
+        export_default_credentials: true
+      name: Gcloud Login
```

Comment:
This step was [failing](https://github.com/solo-io/go-utils/actions/runs/4866590886/jobs/8678320304) even though I have not changed anything. It seems to have worked on [the last PR against main](https://github.com/solo-io/go-utils/pull/509). I know this is not ideal as it pins us to an older version of the action, but my other attempts at fixing this failed and this works.